libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/MapboxNavigationNotification.kt

```diff
@@ -25,13 +25,18 @@ import com.mapbox.services.android.navigation.v5.navigation.notification.Navigat
 import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress
 import com.mapbox.services.android.navigation.v5.utils.DistanceFormatter
 import com.mapbox.services.android.navigation.v5.utils.LocaleUtils
+import com.mapbox.services.android.navigation.v5.utils.extensions.ifNonNull
 import com.mapbox.services.android.navigation.v5.utils.time.TimeFormatter.formatTime
 import java.util.Calendar
 
 /**
  * This is in charge of creating the persistent navigation session notification and updating it.
  */
 internal class MapboxNavigationNotification : NavigationNotification {
+
+    private val END_NAVIGATION_ACTION = "com.mapbox.intent.action.END_NAVIGATION"
```

Comment:
Any particular reason why this was refactor out of the `companion object` and removed `const`?

Reply:
I've answered the question in a prior PR. If values are private to a class there is no need to export them to the outside world via a "companion" object.  In short, a "companion" object exports values and methods to the outside world as public, static values. We should use companions only for that purpose.